content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -175,24 +178,43 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
     $ kubectl get pods --context=$CTX_CLUSTER2 -n istio-system
     {{< /text >}}
 
-1. Update the gateway address in the mesh network configuration:
+1.  Determine the ingress IP and port for `cluster2`.
 
-    * Determine the gateway address for `cluster2`:
+    1.   Set the current context of `kubectl` to `CTX_CLUSTER2`
 
         {{< text bash >}}
-        $ kubectl get --context=$CTX_CLUSTER2 svc --selector=app=istio-ingressgateway -n istio-system -o jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}"
-        169.61.102.93
+        $ export ORIGINAL_CONTEXT=$(kubectl config current-context)
+        $ kubectl config use-context $CTX_CLUSTER2
         {{< /text >}}
 
-    * Edit the istio config map:
+    1.   Follow the instructions in
+        [Determining the ingress IP and ports](/docs/tasks/traffic-management/ingress/#determining-the-ingress-ip-and-ports),
+        to set the `INGRESS_HOST` and `SECURE_INGRESS_PORT` environment variables.
+
+    1.  Restore the previous `kubectl` context:
+
+        {{< text bash >}}
+        $ kubectl config use-context $ORIGINAL_CONTEXT
+        $ unset ORIGINAL_CONTEXT
+        {{< /text >}}
+
+    1.  Print the values of `INGRESS_HOST` and `SECURE_INGRESS_PORT`:
 
         {{< text bash >}}
-        $ kubectl edit cm -n istio-system --context=$CTX_CLUSTER1 istio
+        $ echo ingress host = $INGRESS_HOST
+        $ echo ingress port = $SECURE_INGRESS_PORT
```

Comment:
Should port needs to be 443?

Reply:
No, in case of NodePort it could be a port other than 443.

Reply:
Maybe we can change this to one line. E.g. `$ echo "Ingress gateway of cluster2 - address: $INGRESS_HOST, port: $SECURE_INGRESS_PORT)"`

Because the port var is not used should we just print `$ echo "Ingress gateway address of cluster2: $INGRESS_HOST"`

Reply:
@vadimeisenbergibm ^^^ 
wdyt?

Reply:
Updated it in https://github.com/istio/istio.io/pull/3428/commits/3161593f76f0757b7dfda254847b02b2c6e47f57. Note that port is 31390 in case of NodePort, so it is really used.